### DISCUSSION TITLE
Yolo results being saved in some different directory

### DISCUSSION THREAD

**Discussion by Utsav-2301** [ultralytics/ultralytics#14369] [Q&A]
Hey all, I was using the Asone repository a while ago and as all the computer vision repositories it also had a runs folder where the results were being saved. Now since I'm using the ultralytics repository, there should we a runs folder created in this repository only where the results are getting saved for the scripts running in the current directory, but still the results are getting saved in the Asone runs directory, which is strange but true. What could be the reason for it, is there any global config file due to which its happening or something else

**Comment by glenn-jocher** [comment]
@Utsav-2301 hi there!

Thank you for reaching out. It sounds like there might be some residual configuration from the Asone repository affecting your current setup. Here are a few steps to help resolve this issue:

1. **Check the `project` and `name` Arguments**: When running your scripts, ensure that you are specifying the `project` and `name` arguments to control where the results are saved. For example:
    ```bash
    yolo detect train data=coco128.yaml model=yolov8n.pt epochs=100 imgsz=640 project=ultralytics_runs name=experiment1
    ```

2. **Verify Default Configuration**: The Ultralytics repository uses a default configuration file that might be pointing to the old directory. You can override this by specifying a new configuration file or modifying the existing one. To create a copy of the default configuration, use:
    ```bash
    yolo copy-cfg
    ```
    Then, modify the `default_copy.yaml` file to update the `project` path.

3. **Environment Variables**: Check if there are any environment variables set that might be directing the outputs to the old directory. You can list all environment variables with:
    ```bash
    printenv
    ```

4. **Clear Cache**: Sometimes, clearing the cache can help reset any lingering configurations. You can do this by deleting the cache directory:
    ```bash
    rm -rf ~/.cache/ultralytics
    ```

5. **Update to Latest Version**: Ensure you are using the latest version of the Ultralytics repository. This can help resolve any bugs or issues that have been fixed in recent updates:
    ```bash
    pip install --upgrade ultralytics
    ```

If the issue persists, please provide a minimum reproducible example so we can investigate further. You can find more details on how to create one here: [Minimum Reproducible Example](https://docs.ultralytics.com/help/minimum_reproducible_example).

I hope this helps! If you have any further questions, feel free to ask. 😊
